### DISCUSSION TITLE
Error: Cannot find module '@nomicfoundation/hardhat-toolbox'

### DISCUSSION THREAD

**Discussion by nikhil-b-more** [NomicFoundation/hardhat#3015] [Troubleshooting]
I also tried to install seperately - yarn add @nomicfoundation/hardhat-toolbox.

**Comment by fvictorio** [comment]
If you have it installed this should work. If you can create a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) showing that this happens even with the package installed, I'll take a look.
